ios/versioned-react-native/ABI38_0_0/Expo/ExpoKit/Core/Api/ABI38_0_0EXUpdates.m

```diff
@@ -86,46 +87,24 @@ - (void)sendEventWithBody:(NSDictionary *)body
     reject(@"E_FETCH_UPDATE_FAILED", @"Cannot fetch updates in dev mode", nil);
     return;
   }
-  [_kernelUpdatesServiceDelegate updatesModule:self didRequestManifestWithCacheBehavior:ABI38_0_0EXManifestPrepareToCache success:^(NSDictionary * _Nonnull manifest) {
-    NSString *currentRevisionId = self->_manifest[@"revisionId"];
-    NSString *newRevisionId = manifest[@"revisionId"];
-    if (currentRevisionId && newRevisionId && [currentRevisionId isEqualToString:newRevisionId]) {
+  [_kernelUpdatesServiceDelegate updatesModule:self didRequestBundleWithCompletionQueue:_methodQueue start:^{
+    [self sendEventWithBody:@{ @"type": ABI38_0_0EXUpdatesDownloadStartEventType }];
+  } success:^(NSDictionary * _Nullable manifest) {
+    if (manifest) {
+      [self sendEventWithBody:@{
+        @"type": ABI38_0_0EXUpdatesDownloadFinishedEventType,
+        @"manifest": manifest
+      }];
+      resolve(manifest);
+    } else {
       [self sendEventWithBody:@{ @"type": ABI38_0_0EXUpdatesNotAvailableEventType }];
       resolve(nil);
-      return;
     }
-
-    void (^progressBlock)(NSDictionary * _Nonnull) = ^void(NSDictionary * _Nonnull progressDict) {
-      NSMutableDictionary *eventBody = [progressDict mutableCopy];
-      eventBody[@"type"] = ABI38_0_0EXUpdatesDownloadProgressEventType;
```

Comment:
Do we rely on this progress event anywhere? Might want to add a comment to any users of this to note that we won't get progress events.

Reply:
No, this was never used. The progressBlock parameter exists on EXJavaScriptResource because it's used when loading development mode bundles, but it is/was never fired in production mode so this event was never sent.